libraries/src/Date/Date.php

```diff
@@ -119,6 +119,12 @@ public function __construct($date = 'now', $tz = null)
 		// Force UTC timezone for correct time handling
 		date_default_timezone_set('UTC');
 
+		// If called with null $date
+		if ($date === null)
+		{
+			$date = date('now');
```

Comment:
the parent class expects a string as first parameter
```suggestion
			$date = 'now';
```